test/api/buffer/receivers/delete/expected-etcd-kvs.txt

```diff
@@ -79,8 +79,8 @@
     "id": %d,
     "created": "%s",
     "isSliced": true,
+    "isEncrypted": true,
     %A
-    "federationToken": true
```

Comment:
Z nejakeho dovodu tu nepride `federationToken`, ked `isEncrypted=true`, ale funguje to, neriesim.